ui/app/AppLayouts/Onboarding2/pages/LoginScreen.qml

```diff
@@ -49,7 +49,7 @@ OnboardingPage {
         const hasError = error || detailedError
 
         d.biometricsSuccessful = !hasError
-        d.biometricsFailed = wrongFingerprint
+        d.biometricsFailed = hasError || wrongFingerprint
```

Comment:
Why this change?

Reply:
I think we should mark with red icon whenever there is an error. Currently on mac `wrongFingerprint` will never happen as the request is repeated in the popup whenever wrong fingerprint is provided.